### PR TITLE
feat: add new CLI flag --no-role-detection to parse loose files

### PR DESCRIPTION
**Pull request by ptomas-adacis**
I ran some tests but it is possible that I forgot something.

### CODE REVIEW

File: ansibledoctor/cli.py [hunk 1]
```diff
@@ -68,6 +68,14 @@ def _cli_args(self):
         parser.add_argument(
             "--version", action="version", version="%(prog)s {}".format(__version__)
         )
+        parser.add_argument(
+            "-s",
```
Comment (xoxys) [inline ansibledoctor/cli.py:72]: Lets use `-n`

File: ansibledoctor/cli.py [hunk 2]
```diff
@@ -68,6 +68,14 @@ def _cli_args(self):
...
+            "--skip-role-detection",
```
Comment (xoxys) [inline ansibledoctor/cli.py:73]: According to `-n` I  would like to rename the flag/variables to `--no-role-detection`

File: ansibledoctor/doc_parser.py [hunk 3]
```diff
@@ -76,22 +76,23 @@ def _parse_meta_file(self):
...
-                    ) as e:
-                        message = "{} {}".format(e.context, e.problem)
-                        self.log.sysexit_with_message(
-                            "Unable to read yaml file {}\n{}".format(rfile, message)
-                        )
-
-                    tags_found = nested_lookup("tags", data)
-                    for tag in tags_found:
-                        self._data["tags"][tag] = {}
+        if not self.config.config["skip_role_detection"]:
```
Comment (xoxys) [inline ansibledoctor/doc_parser.py:79]: That's not required. The entire function is not used at the moment.

File: ansibledoctor/doc_parser.py [hunk 4]
```diff
@@ -76,22 +76,23 @@ def _parse_meta_file(self):
...
+                        try:
+                            data = ruamel.yaml.safe_load(yaml_file)
+                        except (
+                            ruamel.yaml.composer.ComposerError, ruamel.yaml.scanner.ScannerError
+                        ) as e:
+                            message = "{} {}".format(e.context, e.problem)
+                            self.log.sysexit_with_message(
+                                "Unable to read yaml file {}\n{}".format(rfile, message)
+                            )
+
```
Comment (xoxys) [inline ansibledoctor/doc_parser.py:92]: You added some white spaces in this line, that's why the CI is complaining :)

**Review by xoxys — CHANGES_REQUESTED** [review]
Thanks for your contribution. Just a few small things to change :)

**Comment by ptomas-adacis** [comment]
Thank you for your checking

**Comment by xoxys** [comment]
What happened? Why have you closed it now?

**Comment by ptomas-adacis** [comment]
Nothing wrong, I did not know that renaming my pull request will cause this closure. I created a new pull request.

**Pull request closed without merging**